react/week1/todo/src/todoapp.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+//import ReactDOM from "react-dom";
+const h2Css = {
+  backgroundColor: "#2a9dbd",
+  padding: "40px",
+  borderRadius: "20px",
+  border: "5px solid black",
+};
+const h1Css = {
+  backgroundColor: "gold",
+  width: "70%",
+  marginLeft: "80px",
+  padding: "30px",
+  borderRadius: "20px",
+};
+
+function RenderTask({ task, task_date }) {
+  return (
+    <>
+      <li>
+        {
+          <h2 style={h2Css}>
+            <span style={{ color: "blue" }}>Task : </span>
+            {task}
+            <span style={{ color: "red", marginLeft: "20px" }}>
+              Deadline :{task_date }
+            </span>
+            {task_date}
+          </h2>
+        }
+      </li>
+    </>
+  );
+}
+
+//<h1>To do application</h1>
```

Comment:
Remove that line. 
Avoid commented code if it is not important to the reviewer.